Add kb article for apple privacy manifest (#830)
telerik/maui-docs: knowledge-base/apple-privacy-manifest-requirement.md

res_type: kb
---

# Apple Privacy Manifest

Apple is introducing new privacy policy for including privacy manifest files in your apps that target iOS, iPadOS, tvOS, visionOS, and watchOS platforms on the app Store. 
For more details on this policy, please review the [Privacy Manifest Files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files) article in the Apple documentation.

> As of May 1st 2024, the Apple privacy policy manifest file must be included in your apps.

## What must be included in the PrivacyInfo.xcprivacy file

The `PrivacyInfo.xcprivacy` file lists:

* The [types of data](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests) collected by your .NET MAUI application or any third-party SDKs.
* The reasons for using certaing [Required Reason API](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api). 

## Add PrivacyInfo.xcprivacy file to your application

Let's review what is needed for your .NET MAUI applicatios when using Telerik .NET MAUI controls:

> 1. Create a `PrivacyInfo.xcprivacy` file inside the `Platforms/iOS` folder. 

Reviewer: ```suggestion
Create a `PrivacyInfo.xcprivacy` file inside the `Platforms/iOS` folder.

For reference, you can use the `PrivacyInfo.xcprivacy` file already added to the [Telerik UI for .NET MAUI Controls Samples]({% slug controls-samples-app %}) and [Telerik UI for .NET MAUI Crypto Tracker]({% slug maui-crypto-app %}) applications:
```

        <dict>
            <key>NSPrivacyAccessedAPIType</key>
            <string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
            <key>NSPrivacyAccessedAPITypeReasons</key>
            <array>
                <string>C617.1</string>
            </array>
        </dict>
        <dict>
            <key>NSPrivacyAccessedAPIType</key>
            <string>NSPrivacyAccessedAPICategorySystemBootTime</string>
            <key>NSPrivacyAccessedAPITypeReasons</key>
            <array>
                <string>35F9.1</string>
            </array>
        </dict>
        <dict>
            <key>NSPrivacyAccessedAPIType</key>
            <string>NSPrivacyAccessedAPICategoryDiskSpace</string>
            <key>NSPrivacyAccessedAPITypeReasons</key>
            <array>
                <string>E174.1</string>
            </array>
        </dict>     
	    <dict>
            <key>NSPrivacyAccessedAPIType</key>
            <string>NSPrivacyAccessedAPICategoryUserDefaults</string>
            <key>NSPrivacyAccessedAPITypeReasons</key>
            <array>
                <string>CA92.1</string>
            </array>
        </dict>
    </array>
</dict>
</plist>
```

## See Also

> * [Microsoft Blogs - Adding Apple Privacy Manifest Support to .NET iOS & .NET MAUI apps](https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/)

Reviewer: ```suggestion
* <a href="https://devblogs.microsoft.com/dotnet/apple-privacy-manifest-support/" target="_blank">Microsoft Blogs - Adding Apple Privacy Manifest Support to .NET iOS & .NET MAUI Apps</a>
```

> * [Apple Documentation - Privacy Manifest Files](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files)

Reviewer: ```suggestion
* <a href="https://developer.apple.com/documentation/bundleresources/privacy_manifest_files" target="_blank">Apple Documentation - Privacy Manifest Files</a>
```

> * [Apple Documentation - Adding a Privacy Manifest File](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/adding_a_privacy_manifest_to_your_app_or_third-party_sdk)

Reviewer: ```suggestion
* <a href="https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/adding_a_privacy_manifest_to_your_app_or_third-party_sdk" target="_blank">Apple Documentation - Adding a Privacy Manifest File</a>
```

> * [Apple Documentation - Describing Data Use in Privacy Manifest File](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests)

Reviewer: ```suggestion
* <a href="https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests" target="_blank">Apple Documentation - Describing Data Use in Privacy Manifests</a>
```

> * [Apple Documentation - Describing Use of Required Reason API](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)

Reviewer: ```suggestion
* <a href="https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api" target="_blank">Apple Documentation - Describing Use of Required Reason API</a>
```
